impl/torch/helper.hpp

```diff
@@ -53,11 +53,23 @@ namespace impl {
 
 namespace aten {
 
+
+static const size_t MAX_GPU_NUMS = 32;
```

Comment:
```suggestion
constexpr size_t MAX_GPU_NUMS = 16;
```
而且也可以放到函数里。

貌似最大值是 16？dipu 里面是。

Reply:
> 而且也可以放到函数里。
> 
> 貌似最大值是 16？dipu 里面是。

这个我调整下。